aptos-move/framework/aptos-framework/sources/reconfiguration.spec.move

```diff
@@ -67,7 +67,9 @@ spec aptos_framework::reconfiguration {
         use aptos_framework::transaction_fee;
         use aptos_framework::staking_config;
 
-        pragma verify_duration_estimate = 120; // TODO: set because of timeout (property proved)
+        //We use aborts_if_is_partial to avoid timeout
+        //All properties is verifed
+        pragma aborts_if_is_partial;
```

Comment:
Why did you change it to `aborts_if_is_partial`? Any problem with `verify_duration_estimate`?